webapp/channels/src/components/user_group_popover/group_member_list/group_member_list.tsx

```diff
@@ -171,13 +179,20 @@ const GroupMemberList = (props: Props) => {
                         hideStatus={user.is_bot}
                     >
                         <UserButton>
-                            <Avatar
-                                username={user.username}
-                                size={'sm'}
-                                url={Utils.imageURLForUser(user?.id ?? '')}
-                                className={'avatar-post-preview'}
-                                tabIndex={-1}
-                            />
+                            <span className='status-wrapper'>
+                                <Avatar
+                                    username={user.username}
+                                    size={'sm'}
+                                    url={Utils.imageURLForUser(user?.id ?? '')}
+                                    className={'avatar-post-preview'}
+                                    tabIndex={-1}
+                                />
+                                <StatusIcon
+                                    className='status user-popover-status'
```

Comment:
Maybe I am missing something, but the only place `user-popover-status` is defined, is in `webapp/channels/src/components/profile_popover/profile_popover.scss`, and it is defined under the `user-popover-image` class, so... it should not be applying any stile here, right? If so, why the class name?

Reply:
@larkox you're right the props classname and button are useless in these cases, I thought I had to add a style for the status icon but there's already a `.status` included in `.status-wrapper` class so no need to add it, thanks for the feedback